packages/cms-data-sync/test/events/events.data-migration.test.ts

```diff
@@ -0,0 +1,582 @@
+        url: 'https://drive.google.com/drive/folders/id',
+      },
+      { title: 'Slack', url: 'https://slack.com/intl/pt-br/' },
+    ],
+  },
+  meetingMaterialsPermanentlyUnavailable: { 'en-US': null },
+  notes: {
+    'en-US': {
+      content: [
+        {
+          content: [{ data: {}, marks: [], nodeType: 'text', value: 'notes' }],
+          data: {},
+          nodeType: 'paragraph',
+        },
+      ],
+      data: {},
+      nodeType: 'document',
+    },
+  },
+  notesPermanentlyUnavailable: { 'en-US': null },
+  notesUpdatedAt: { 'en-US': '2023-05-17T13:38:51.128Z' },
+  presentation: { 'en-US': null },
+  presentationPermanentlyUnavailable: { 'en-US': null },
+  presentationUpdatedAt: { 'en-US': null },
+  speakers: {
+    'en-US': [
+      { sys: { id: 'entry-id', linkType: 'Entry', type: 'Link' } },
+      { sys: { id: 'entry-id', linkType: 'Entry', type: 'Link' } },
+    ],
+  },
+  tags: { 'en-US': ['googleMeet', 'science'] },
+  thumbnail: { 'en-US': undefined },
+  videoRecording: { 'en-US': null },
+  videoRecordingPermanentlyUnavailable: { 'en-US': null },
+  videoRecordingUpdatedAt: { 'en-US': null },
+};
+
+const baseCreatePayload = {
+  ...baseUpdatePayload,
+  calendar: {
+    'en-US': {
+      sys: {
+        id: '1e34bfaa-6752-41ec-a796-eca801228a90',
+        linkType: 'Entry',
+        type: 'Link',
+      },
+    },
+  },
+  description: { 'en-US': null },
+  endDate: { 'en-US': '2023-04-03T17:00:00Z' },
+  endDateTimeZone: { 'en-US': 'America/Sao_Paulo' },
+  eventLink: { 'en-US': null },
+  googleId: { 'en-US': '1jhm4181bs6ck0esegje4nf6ur' },
+  startDate: { 'en-US': '2023-04-03T16:00:00Z' },
+  startDateTimeZone: { 'en-US': 'America/Sao_Paulo' },
+  status: { 'en-US': 'Confirmed' },
+  title: { 'en-US': 'Amazing event!!!' },
+};
+
+describe('Migrate events', () => {
+  let contentfulEnv: Environment;
+  let squidexGraphqlClientMock: jest.Mocked<SquidexGraphqlClient>;
+
+  const consoleLogRef = console.log;
+  beforeEach(() => {
+    console.log = jest.fn();
+    contentfulEnv = getContentfulEnvironmentMock();
+    squidexGraphqlClientMock = {
+      request: jest.fn(),
+    };
+
+    (getSquidexAndContentfulClients as jest.Mock).mockResolvedValueOnce({
+      contentfulEnvironment: contentfulEnv,
+      squidexGraphqlClient: squidexGraphqlClientMock,
+    });
+
+    jest.spyOn(contentfulEnv, 'getEntries').mockResolvedValueOnce({
+      total: 1,
+      items: [eventEntry],
+      skip: 0,
+      limit: 10,
+      toPlainObject: jest.fn(),
+      sys: { type: 'Array' },
+    });
+    jest.spyOn(contentfulEnv, 'getEntry').mockResolvedValue(getEntry({}));
+    jest.spyOn(contentfulEnv, 'createEntry').mockResolvedValue(getEntry({}));
+
+    jest.spyOn(eventEntry, 'publish').mockResolvedValue(getEntry({}));
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  afterAll(() => {
+    console.log = consoleLogRef;
+  });
+
+  it('does not call clear contentful entries for events', async () => {
```

Comment:
maybe let's enable clearing previous entries conditionally? i feel like by default we should dump them all anyway, at the moment i dont see how we would need to NOT do that at all